packages/sui-studio/package.json

```diff
@@ -33,7 +33,7 @@
     "bundle-loader": "0.5.6",
     "chai": "4.2.0",
     "chai-dom": "1.8.1",
-    "classnames": "2.2.6",
+    "classnames": "2.2.5",
```

Comment:
We want to downgrade the `classnames` package?

Reply:
Yes, to be sure we're using the same across. In other PR we could upgrade the classnames version in all the packages to be sure we load it only once.